src/main/scala/Schema.scala

```diff
@@ -0,0 +1,62 @@
+package CSVValidation
+
+import com.fasterxml.jackson.databind.{JsonNode, ObjectMapper}
+import com.fasterxml.jackson.databind.node.ObjectNode
+import traits.JavaIteratorExtensions.IteratorHasAsScalaArray
+
+import java.net.URL
+import java.nio.file.Paths
+object Schema {
+  val objectMapper = new ObjectMapper()
+
+  def loadMetadataAndValidate(
+      uri: String,
+      outputErrors: Boolean = true
```

Comment:
What's `outputErrors` used for?

Reply:
It might be better to use the `Either<TableGroup, String>` paradigm here. You get either a table group or a string representing the error.

Reply:
The output errors thing was a feature in csvlint and tried to implement the same in scala as well (though it was not complete as the scopt OParser didn't had the option to pass this flag in). Now I have removed all the references related to this flag and warnings will be printed out by default. Can bring this feature back in when required.

Reply:
I have used the Either thing as you suggested